src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1588,6 +1588,10 @@ RESULT_CODE ProtocolHandlerImpl::HandleControlMessageStartSession(
       std::find(audio_transports.begin(), audio_transports.end(), transport) !=
           audio_transports.end();
 
+  service_status_update_handler_->OnServiceUpdate(
+      session_observer_.KeyFromPair(
```

Comment:
@mked-luxoft extract temporary const variables

Reply:
@LuxoftAKutsan fixed, and rebased on newest feature branch